### ISSUE TITLE
1.9 syntax used

### ISSUE THREAD

**Issue by elia** [lucasefe/themes_for_rails#53]
You should declare a 1.9 only compatibility in the gem spec or (preferably) make it 1.8 friendly

The "offending" line is this:

```
SyntaxError: /Users/elia/Code/Mikamai/pradagroup-html/vendor/gems/themes_for_rails/lib/themes_for_rails/action_view.rb:48: syntax error, unexpected tIDENTIFIER, expecting tAMPER
      javascript_include_tag(*files, options)
                                            ^
```

**Comment by SeanRoberts** [comment]
+1

**Comment by lucasefe** [comment]
Released! gem install themes_for_rails -v 0.5.1
